app/Repositories/Task/TaskRepository.php

```diff
@@ -27,4 +28,43 @@ public function getTasks($request)
             'data' => $tasks->paginate($request['perPage'])
         ];
     }
+
+    public function createTask()
+    {
+        $subject_id = [1, 2, 3, 5];
+        $task = [
+            'name' => 'test',
+            'description' => 'test',
+            'content' => 'test',
+            'is_active' => 1
+        ];
+        try {
+            $task = $this->model->create($task);
+            $task->subjectTask()->attach($subject_id, ['status' => 1]);
+        } catch (Exception $e) {
+            return [
+                'success' => false,
+                'message' => $e->getMessage()
+            ];
+        }
+
+        return [
+            'success' => true
+        ];
+    }
+
+    public function showTask($id)
+    {
+        try {
+            return [
+                'success' => true,
+                'data' => $this->model->findOrFail($id)
+            ];
+        } catch (Exception $e) {
+            return [
+                'success' => false,
+                'message' => $e->getMessage()
+            ];
+        }
```

Comment:
tạo 1 biến lưu $this->model->findOrFail($id) lại dưới data sẽ trả về biến đó! k nên viết tắt kiểu này